app/components/dashboard/DcChart.vue

```
@@ -6125,26 +6085,27 @@ const initChartDate2Stacks = (chartDateW, stackOn) => {
   // スタック登録 - CHART_DATE_STACK_GRP[0]
   for (var no = 0; no < mm.chartStack[STACK_CND].length; no++) {
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)